aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/status/DBInstanceStatus.java

```diff
@@ -9,10 +9,15 @@ public enum DBInstanceStatus implements TerminableStatus {
     Deleting("deleting"),
     Failed("failed", true),
     InaccessibleEncryptionCredentials("inaccessible-encryption-credentials", true),
+    InaccessibleEncryptionCredentialsRecoverable("inaccessible-encryption-credentials-recoverable", true),
```

Comment:
```
The KMS key used to encrypt or decrypt the DB instance can't be accessed. However, if the KMS key is active, restarting the DB instance can recover it.
```
Is a `reboot-db-instance` service call required to move the instance out of this state? We need to consider if we are inclined to account for this WF or if we prefer to delegate this resolution elsewhere.